### PR TITLE
add orientation warning to acc calibration page

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-accelerometer-calibration.rst [hunk 1]
```diff
@@ -16,6 +16,8 @@ This article shows how to perform basic accelerometer calibration (using
 Calibration steps
 =================
 
+.. warning:: Be sure the :ref:`AHRS_ORIENTATION<AHRS_ORIENTATION>` is correct before doing the calibration!
```
Comment (rmackay9) [inline common/source/docs/common-accelerometer-calibration.rst]: let's add a warning to this page too: http://ardupilot.org/copter/docs/common-mounting-the-flight-controller.html

Comment (rmackay9) [reply common/source/docs/common-accelerometer-calibration.rst]: let's remove "in Mission Planner" from the title.
Let's remove the note, "Accelerometer calibration is mandatory if you’re using EKF to estimate vehicle position."

Comment (rmackay9) [reply common/source/docs/common-accelerometer-calibration.rst]: maybe let's change the warning to make it more clear that users only need to worry about the orientation if it's non-standard.

If the board is mounted in a non-standard orientation (i.e. arrow is *not* pointing forward) then please ensure the AHRS orientation is set before doing the accelerometer calibration.

Comment (Hwurzburg) [reply common/source/docs/common-accelerometer-calibration.rst]: done

**Comment by Hwurzburg** [comment]
Merged with changes

**Pull request merged**